README.md

```diff
@@ -47,7 +47,7 @@ Options:
 3. Submit Pull Request with comprehensive description of changes
 
 ## Link to the Video
-[![Video](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)
+- Click here to watch the Video [![Video](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)](https://www.loom.com/share/cc0c095b63c1428489cd77454c7948b2)
```

Comment:
- [ ] Hey @Josphat205 , great job implementing all of the required changes, I love your design and the animation effects are awesome. But as the previous Reviewer said, your video should not exceed 5 minutes. This is good practice for interviews where you have to explain your point of view on the matter in a limited amount of time (usually 3 minutes) so please make sure your video does not exceed 5 minutes.